SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/GameModeReplicationTest/GameModeReplicationTest.h

```diff
@@ -13,7 +13,7 @@
 
 #include "GameModeReplicationTest.generated.h"
 
-UCLASS(BlueprintType)
+UCLASS(BlueprintType, HideDropdown)
```

Comment:
@improbable-dmitrii do you remember why all of these are blueprint types?

Reply:
An artifact of manual functional test map creation, can be removed now.

Reply:
removing